integration_tests/projects/008_pure_python_models/scripts/post_hook.py

```diff
@@ -2,6 +2,7 @@
 from functools import reduce
 import os
 import time
+from fal.context import *
```

Comment:
and everything still works if you dont import this right?

Reply:
yep!